src/main/scala/fundamentals/level02/TypesExercises.scala

```diff
@@ -158,12 +202,25 @@ object TypesExercises {
 
   def showTrafficLight(trafficLight: TrafficLight): String = ???
 
+  /**
+    * *********************************************************
+    * * Section 5.1 - TDD again and extend the function *******
+    * *********************************************************
+    */
+
   /**
     * Now introduce a new type of `TrafficLight` called `Flashing`.
     *
-    * What happens when you try to compile now?
+    * 1. Fill in the unit test for this new scenario: "showTrafficLight should show Flashing"
```

Comment:
Step 2 first, then 1, 3 and 4.

Leave out "How to compile" because that's too specific to IntelliJ.